curation_concerns-models/app/models/concerns/curation_concerns/generic_file/derivatives.rb

```diff
@@ -0,0 +1,33 @@
+module CurationConcerns
+  module GenericFile
+    module Derivatives
+      extend ActiveSupport::Concern
+
+      included do
+        include Hydra::Derivatives
+        # This was taken directly from Sufia's GenericFile::Derivative.
+        # Using File and ServiceFile so this class will not generate two derivatives 
+        # with the same RDF URI. Is there a mime type ontology we should be using instead?
+        directly_contains_one :ogg, through: :files, type: ::RDF::URI("http://pcdm.org/use#ServiceFile"), class_name: "Hydra::PCDM::File"
```

Comment:
Sorry, my comments should have been spaced out better, didn't take those four lines from sufia. They are just following the pattern used in Hydra::Works to declare derivatives. Since where this should ultimately lives is still in flux @tampakis and I discussed it and made it part of in the test app installer which mixes it into GenericFile to keep the test coverage active in this gem.